File: easy/119 (Pascal's Triangle II)/solution.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<vector<int>> ans;
        for(int i=0; i<=rowIndex; i++){
            ans.push_back(vector<int>(i+1,1));
        }
        for(int i=1; i<=rowIndex; i++){
            for(int j=1; j<ans[i].size() - 1; j++){
                ans[i][j] = ans[i-1][j] + ans[i-1][j-1];
            } 
        }
        return ans[rowIndex];
    }
};

File: easy/119 (Pascal's Triangle II)/solution.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        l = []
        for i in range(0, (rowIndex + 1)):
            l.append([])
            l[i].append(1)
            for y in range(1, i):
                l[i].append(l[i - 1][y] + l[i - 1][y - 1])
            if(i > 0): l[i].append(1)
        
        return l[rowIndex]